docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -40,7 +46,7 @@ assets = kili.assets("<your_project_id>", fields=["externalId", "latestLabel.jso
 print(assets[0])
 {'latestLabel': {'jsonResponse': {'CLASSIFICATION_JOB': {'categories': [{'name': 'VEHICLE'}]}}}, 'externalId': '0'}
 ```
-  3. You can now get your label this way, and for example write the category name into a `0.txt` file:
+  3. You can now get your label, for example, write the category name into a `0.txt` file:
```

Comment:
You can now get your label, and write the category name into a `0.txt` file for example:

Reply:
done